src/Tizen.GenericTextClassifier/Tizen.GenericTextClassifier/GenericTextClassifier.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.InteropServices;
+
+namespace Tizen.GenericTextClassifier
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct CategoryListStruct
+    {
+        internal char[] first_category;
+        internal char[] second_category;
+        internal char[] third_category;
+        internal float first_category_score;
+        internal float second_category_score;
+        internal float third_category_score;
+    }
+
+    class GenericTextClassifier
+    {
+        internal const string LogTag = "Tizen.GenericTextClassifier";
+        /*int text_classifier_initialize(void);*/
+        public int TextClassifierInitialize()
+        {
+            int ret;
+
+            ret = Interop.GTCAPI.TextClassifierInitialize();
+
+            if (ret != (int)GenericTextClassifierError.None)
+            {
+                Log.Error(LogTag, "Failed to Initialize - " + (GenericTextClassifierError)ret);
+                GenericTextClassifierErrorFactory.ThrowGenericTextClassifierException(ret);
+            }
+
+            return ret;
+        }
+        public int TextClassifierDeinitialize()
+        {   
+            int ret;
+
+            ret = Interop.GTCAPI.TextClassifierDeinitialize();
+
+            if (ret != (int)GenericTextClassifierError.None)
+            {
+                Log.Error(LogTag, "Failed to Deinitialize - " + (GenericTextClassifierError)ret);
+                GenericTextClassifierErrorFactory.ThrowGenericTextClassifierException(ret);
+            }
+
+            return ret;
+        }
+
+        public int TextClassifierCreate(IntPtr classifier_handle, String app_id, String user_id)
```

Comment:
Please use camelCase for method arguments instead of underscore.
```suggestion
        public int TextClassifierCreate(IntPtr classifierHandle, String appId, String userId)
```